doc/introduction/interfaces/jax.rst

```diff
@@ -0,0 +1,148 @@
+    def circuit1(phi, theta):
+        qml.RX(phi[0], wires=0)
+        qml.RY(phi[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+        qml.PhaseShift(theta, wires=0)
+        return qml.expval(qml.PauliZ(0)), qml.expval(qml.Hadamard(1))
+
+The QNode ``circuit1()`` is now a JAX-capable QNode, accepting ``jax.DeviceArray`` objects
+as input, and returning ``jax.DeviceArray`` objects. It can now be used like any other JAX function:
+
+>>> phi = jnp.array([0.5, 0.1])
+>>> theta = jnp.array(0.2)
+>>> circuit1(phi, theta)
+DeviceArray([0.8776, 0.6880], dtype=float64)
+
+Quantum gradients using JAX
+---------------------------
+
+Since a JAX-interfacing QNode acts like any other JAX interfacing python function,
+the standard method used to calculate gradients with JAX can be used.
+
+For example:
+
+.. code-block:: python
+
+    dev = qml.device('default.qubit.jax', wires=2)
+
+    @qml.qnode(dev, interface='jax')
+    def circuit3(phi, theta):
+        qml.RX(phi[0], wires=0)
+        qml.RY(phi[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+        qml.PhaseShift(theta, wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+    phi = jnp.array([0.5, 0.1])
+    theta = jnp.array(0.2)
+    grads = jax.grad(circuit3, argnums=(0, 1))
+    phi_grad, theta_grad = grads(phi, theta)
+
+This has output:
+
+>>> phi_grad
+DeviceArray([-0.47942555,  0.        ], dtype=float32)
+>>> theta_grad
+DeviceArray(-3.4332792e-10, dtype=float32)
+
+
+.. _jax_optimize:
+
+Using jax.jit on QNodes
+-----------------------
+
+To fully utilize the power and speed of JAX, you'll need to just-in-time compile
+your functions - a process called ``jitting``. 
+If you're only taking expectation values, you can simply add the decorator on your QNode directly.
+
+.. code-block:: python
+
+    dev = qml.device('default.qubit.jax', wires=2)
+    
+    @jax.jit  # QNode calls will now be jitted, and should run faster.
+    @qml.qnode(dev, interface='jax')
+    def circuit4(phi, theta):
+        qml.RX(phi[0], wires=0)
+        qml.RZ(phi[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+        qml.RX(theta, wires=0)
+        return qml.expval(qml.PauliZ(0))
+
+
+Randomness: Shots and Samples
+-----------------------------
+In JAX, there is no such thing as statefull randomness, meaning all random number generators must be explicitly seeded. 
+(See the `JAX random package documentation <https://jax.readthedocs.io/en/latest/jax.random.html?highlight=random#module-jax.random>`_ for more details).
+While a real quantum computer's randomness can not be seeded, we are forced to when using a JAX simulator.
+
+If you want to simulate randomness (meaning you either have a finite ``shots`` value, or the qnode returns ``qml.samples()``), JAX requires you to use a ``jax.random.PRNGKey``. On a normal method, we handle this automatically for you. However, if you wish to decorate your qnode with a ``jax.jit``, you must contruct your device inside of the ``jax.jit`` decorated method.

```

Comment:
This is really hard to state. What do you think about this?
```suggestion
When simulations include randomness (i.e., if the device has a finite ``shots`` value, or the qnode returns ``qml.samples()``), JAX requires a ``jax.random.PRNGKey``. Usually, PennyLane automatically handles this for you. However, if you wish to use jitting, not only the qnode, but also the device needs to be created in the context of the ``@jax.jit`` decorator. This can be achieved by wrapping device and qnode creation into a function:

is constructed inside a function that carries .
```